### ISSUE TITLE
Cannot create object with accented characters (following ECMA 5.1 compliance)

### ISSUE THREAD

**Issue by imsmart-tech** [Taritsyn/JavaScriptEngineSwitcher#36]
This is using jint, but I suspect it's the same with other engines.
For example, this works:

`script._engine.Execute("var À_la_maison = 5;");`

but this does not work:

```
int test= 5;
engine.EmbedHostObject("À_la_maison", test);
```

it returns error:

> "The script item name 'À_la_maison' has incorrect format."

Of course, without accent it works well.

**Comment by imsmart-tech** [comment]
This works fine, using jint directly without JavascriptEngineSwitcher:

```
            var p = new Person
            {
                Name = "Mickey Mouse"
            };

            var engine = new Engine()
                .SetValue("À_la_maison", p)
                .Execute("À_la_maison.Name === 'Mickey Mouse'")
                ;

```

**Comment by Taritsyn** [comment]
Hello, Paul!

This error fixed in [version 2.4.9](https://github.com/Taritsyn/JavaScriptEngineSwitcher/releases/tag/v2.4.9).

When you deal with primitive types, then you should use the `SetVariableValue` method instead of the `EmbedHostObject` method:

```csharp
int test= 5;
engine.SetVariableValue("À_la_maison", test);
```
